### DISCUSSION TITLE
OpenStore

### DISCUSSION THREAD

**Discussion by emersonlaurentino** [vtex/faststore#672] [General]
I and @tlgimenes were talking and mapping out how we want the `openstore` to be.

What do you think about it?

[Excalidraw](https://excalidraw.com/#json=5952115860045824,tlIyEP5_p2rzGTJz9NE9SQ) to suggestions

![open-store](https://user-images.githubusercontent.com/10627086/117878000-a4c49400-b27b-11eb-9eb5-b1ae7d1745d9.png)

**Comment by klzns** [comment]
Somethings to add:
- postal code API
- product recommendations
- wish list
- product reviews

**Comment by tlgimenes** [comment]
One thing I'm really not sure @klzns is the name of the broadcast API. Today we use Pixel, but I'm not sure we people use this naming scheme outside VTEX and Facebook

**Reply by emersonlaurentino** [reply]
I think we should be use Broadcast, is what makes the most sense to me

**Reply by tlgimenes** [reply]
Apparently this is a known industry term: Looking for Tracking pixel on google presents lots of results, including this one: https://en.ryte.com/wiki/Tracking_Pixel

**Reply by emersonlaurentino** [reply]
hmm, truly. Maybe we should keep Pixel. 😞

**Reply by klzns** [reply]
I think Analytics is a good name too
